indexing-hadoop/src/main/java/io/druid/indexer/hadoop/DatasourceIngestionSpec.java

```diff
@@ -108,19 +112,66 @@ public QueryGranularity getGranularity()
     return metrics;
   }
 
+  @JsonProperty
+  public boolean getIgnoreWhenNoSegments()
```

Comment:
thought the PR has ser desr UTs, @xvrl you think it is not enough and it may fail ?